client/signup/step-wrapper/index.jsx

```diff
@@ -71,7 +71,7 @@ class StepWrapper extends Component {
 						flowName={ this.props.flowName }
 						stepName={ this.props.stepName }
 						labelText={ this.props.skipLabelText }
-						cssClass={ !! this.props.skipHeadingText && ' navigation-link--has-skip-heading ' }
+						cssClass={ this.props.skipHeadingText && 'navigation-link--has-skip-heading' }
```

Comment:
Unrelated change that was causing propType warnings on the first step when `skipHeadingText` was being cast to a bool of `false`.